llvm/test/tools/sycl-post-link/spec-constants/SYCL-2020-zeroinitializer-array-of-arrays.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: sycl-post-link --spec-const=native -S < %s -o %t.files.table
 ; RUN: FileCheck %s -input-file=%t.files_0.ll --check-prefix CHECK-IR
 ; RUN: FileCheck %s -input-file=%t.files_0.prop --check-prefix CHECK-PROP
-; RUN: sycl-post-link -debug-only=SpecConst --spec-const=native -S < %s 2>&1 | FileCheck %s --check-prefix=CHECK-LOG
+; RUN: %if asserts %{ sycl-post-link -debug-only=SpecConst --spec-const=native -S < %s 2>&1 | FileCheck %s --check-prefix=CHECK-LOG %} %else %{ %}
```

Comment:
As we don't test "else" branch of this condition, it's better to put `; REQUIRES: asserts`. LIT will report accurate results in this case.

Reply:
As far as I understand, `REQUIRES: asserts` would affect to the whole test (all the `RUN` lines), I only want it to affect the one line.